raster/src/main/scala/geotrellis/raster/VectorToRaster.scala

```diff
@@ -84,14 +81,52 @@ object VectorToRaster {
   def idwInterpolate(points: Seq[PointFeature[Int]], re: RasterExtent): Tile =
     idwInterpolate(points, re, None)
 
+  def idwInterpolateDouble(points: Seq[PointFeature[Double]], re: RasterExtent): Tile =
+    idwInterpolateDouble(points, re, None)
+
   /**
     * Compute an Inverse Distance Weighting raster over the given
     * extent from the given set known-points.  Please see
     * https://en.wikipedia.org/wiki/Inverse_distance_weighting for
     * more details.
+    *
+    * @param   points  A collection of known-points
+    * @param   re      The study area
+    * @param   radius  Interpolation radius in cells
+    * @return          The data interpolated across the study area
     */
-  def idwInterpolate(points: Seq[PointFeature[Int]], re: RasterExtent, radius: Int): Tile =
+  def idwInterpolate(points: Seq[PointFeature[Int]], re: RasterExtent, radius: Circle): Tile = {
```

Comment:
This is a convenient overload, but I don't think the math is correct. `Circle(1)` defines a 3x3 cell `Neighborhood`, which means the radius should be 1.5 cell width, to account for half a cell from the center point.